README.md

```diff
@@ -19,9 +19,18 @@ To access libraries and schemas from the HERE platform, you need a HERE Workspac
 Once you have enabled your account you need to create the credentials and prepare your environment. Workspace users can find corresponding guidance [in the documentation for Java and Scala developers]( https://developer.here.com/olp/documentation/sdk-developer-guide/dev_guide/topics/how-to-use-sdk.html). Marketplace users can find instructions [in the Marketplace Consumer user guide](https://developer.here.com/olp/documentation/marketplace-consumer/user-guide/topics/get_catalog_data.html#register-app).
 
 Please note, by default the Maven Wagon plugin uses the `credentials.properties` file provided in the `.here` directory in the user home directory. 
-There are two options to override the path:
+There are three options to override the credentials:
 - The first option is the system property `hereCredentialsFile`, the property should be added to the maven command the following way `-DhereCredentialsFile=/full/path/to/credentials.properties`.
-- The second option is the environment variable `HERE_CREDENTIALS_FILE`.  The variable should contain the full file path to the `credentials.properties` file to be used. The variable is taken into account only if there is no system property provided. 
+- The second option is the environment variable `HERE_CREDENTIALS_FILE`.  The variable should contain the full file path to the `credentials.properties` file to be used. The variable is taken into account only if there is no system property provided.
+- The third option is the system property `hereCredentialsString`, the property should be added to the maven command the following way:
```

Comment:
I would put it as first item. So that the items go from higher priority to lower.